cli/consumers.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	mfxsdk "github.com/mainflux/mainflux/pkg/sdk/go"
+	"github.com/spf13/cobra"
+)
+
+var cmdSubscription = []cobra.Command{
+	{
+		Use:   "create <topic> <contact> <user_auth_token>",
+		Short: "Create subscription",
+		Long:  `Create new subscription`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) != 3 {
+				logUsage(cmd.Use)
+				return
+			}
+
+			id, err := sdk.CreateSubscription(args[0], args[1], args[2])
```

Comment:
probably best to name these args[i]  variables to more descriptive names for readability,

Reply:
That would make it spaghetti code